### DISCUSSION TITLE
如何使用nacos config来连接数据库

### DISCUSSION THREAD

**Discussion by KingDracula** [midwayjs/midway#3740] [Q&A] [answered]
### Describe the problem（描述问题）

我使用的是Sequelize 的Repository 模式操作数据库，但是想要使用nacos config来进行数据库的配置，我尝试在onConfigLoad的方法中读取nacos来更改数据库配置，但是发现会在执行这个方法之前初始化Sequelize配置。
请教一下这个应该要怎么读取设置数据库的配置

### Midway Versions（Midway 版本）

  midway-version@1.3.8

**Comment by czy88840616** [comment, marked as answer]
1、不要在 onConfigLoad 调用任意数据库的代码避免提前初始化，包括 inject
2、在 onConfigLoad 中直接用 nacos getConfig 获取配置

**Reply by KingDracula** [reply]
已经解决了，多谢了
